jobs/furnishings/requirements.txt

```diff
@@ -31,5 +31,7 @@ six==1.15.0
 urllib3==1.26.11
 requests==2.25.1
 cachelib==0.9.0
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+# git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+git+https://github.com/chenhongjing/lear.git@22827-sql-versioning#egg=legal_api&subdirectory=legal-api
 git+https://github.com/bcgov/business-schemas.git@2.15.38#egg=registry_schemas
+git+https://github.com/chenhongjing/lear.git@22827-sql-versioning#egg=sql-versioning&subdirectory=python/common/sql-versioning
```

Comment:
Temporarily update these dependencies to use the version in my branch to make sure CI pass and no other issues. Will update them to main branch before the code is merged.